<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Select Items</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Items</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="itemClearButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>501</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Items">
      <attribute name="title">
       <string>Items</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
       </property>
       <property name="title">
        <string>Filter:</string>
       </property>
       <widget class="QComboBox" name="groupList">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>171</width>
          <height>26</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Select group</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Book</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video Games</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sports</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N/A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Baby Product</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Software</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Toy</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
       <widget class="QComboBox" name="minItemAvgRating">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>171</width>
          <height>26</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Select minimum rating</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.0</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Avg. Rating</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>191</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
       </property>
       <property name="title">
        <string>Sort by:</string>
       </property>
       <widget class="QComboBox" name="itemsSortBy">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Sort By</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alphabetically A-Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alphabetically Z-A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Average Rating</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Salesrank</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Similarity</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="itemRecButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Get Item Recommendations</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Users">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
       </property>
       <property name="title">
        <string>Filter:</string>
       </property>
       <widget class="QComboBox" name="minUserAvgRating">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>171</width>
          <height>26</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Select minimum rating</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.0</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="userMinReviews">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>80</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Min. Reviews</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Avg. Rating</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>191</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
       </property>
       <property name="title">
        <string>Sort by:</string>
       </property>
       <widget class="QComboBox" name="userSortBy">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>161</width>
          <height>26</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Sort By</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alphabetically A-Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alphabetically Z-A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Average Rating</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Number of Reviews</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Similarity</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="userRecButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Get User Recommendations</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QListWidget" name="algList">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string>Cosine Similarity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pearson Similarity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jaccard Similarity</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Algorithm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select output type:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>391</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QPushButton" name="clearAllButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>480</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
    <widget class="QLabel" name="metricsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="outputList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>351</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
